zebra-consensus/src/transaction/tests.rs

```diff
@@ -451,6 +454,86 @@ async fn v5_transaction_with_transparent_transfer_is_rejected_by_the_script() {
     );
 }
 
+/// Tests transactions with Sprout transfers.
+///
+/// This is actually two tests:
+/// - Test if signed V4 transaction with a dummy [`sprout::JoinSplit`] is accepted.
+/// - Test if an unsigned V4 transaction with a dummy [`sprout::JoinSplit`] is rejected.
+///
+/// The first test verifies if the transaction verifier correctly accepts a signed transaction. The
+/// second test verifies if the transaction verifier correctly rejects the transaction because of
+/// the invalid signature.
+///
+/// These tests are grouped together because of a limitation to test shared [`tower_batch::Batch`]
+/// services. Such services spawn a Tokio task in the runtime, and `#[tokio::test]` can create a
+/// separate runtime for each test. This means that the worker task is created for one test and
+/// destroyed before the other gets a chance to use it.
```

Comment:
Can you reference the runtime fix ticket here?

Reply:
```suggestion
/// These tests are grouped together because of a limitation to test shared [`tower_batch::Batch`]
/// services. Such services spawn a Tokio task in the runtime, and `#[tokio::test]` can create a
/// separate runtime for each test. This means that the worker task is created for one test and
/// destroyed before the other gets a chance to use it. (We'll fix this in #2390.)
```